### PR TITLE
Pass the filter info in aggregate operator from jvm to native

### PR DESCRIPTION
**Pull request by JkSelf**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

**Review by jinchengchenghh — APPROVED** [review]

**Comment by jinchengchenghh** [comment]
Wait CI fix

**Comment by rui-mo** [comment]
This PR https://github.com/oap-project/gluten/pull/816 would help solve the CH backend issue.

File: gluten-ut/src/test/scala/org/apache/spark/sql/GlutenSQLQueryTestSuite.scala [hunk 1]
```diff
@@ -209,7 +209,26 @@ class GlutenSQLQueryTestSuite extends QueryTest with SharedSparkSession with SQL
   /** List of supported cases to run, in lower case.
    * Please add to the supported list after enabling a sql test.
    */
-  private val supportedList: Set[String] = Set()
+  private val supportedList: Set[String] = Set(
```
Comment (rui-mo) [inline gluten-ut/src/test/scala/org/apache/spark/sql/GlutenSQLQueryTestSuite.scala:212]: Please do a rebase for the supported list, thanks.

**Review by rui-mo — APPROVED** [review]

File: gluten-ut/src/test/scala/org/apache/spark/sql/GlutenSQLQueryTestSuite.scala (hunk 1 above)
Comment (JkSelf) [reply gluten-ut/src/test/scala/org/apache/spark/sql/GlutenSQLQueryTestSuite.scala]: Rebased.

File: backends-clickhouse/src/main/scala/io/glutenproject/execution/CHHashAggregateExecTransformer.scala [hunk 2]
```diff
@@ -227,8 +227,16 @@ case class CHHashAggregateExecTransformer(
       }
     }
 
+    val aggFilterList = new util.ArrayList[ExpressionNode]()
```
Comment (JkSelf) [inline backends-clickhouse/src/main/scala/io/glutenproject/execution/CHHashAggregateExecTransformer.scala:230]: @zzcclp 
Velox backend need to pass the filter info in aggregate function from jvm to native. I also update the related code in ch backend to fix the compile errors. Can you help to review? Thanks.

Comment (JkSelf) [reply backends-clickhouse/src/main/scala/io/glutenproject/execution/CHHashAggregateExecTransformer.scala:230]: Merged into main branch firstly. If any issue, I will filed another PR to fix. Thanks.

**Pull request merged**